input_data.go

```diff
@@ -6,7 +6,9 @@ type Modifier int
 
 var (
```

Comment:
You make a valid point. 

It is in fact meant to be exported. I'm not sure what the new name of `mouse` would become. 

Perhaps move `engo.Mouse` to `engo.Input.Mouse`? Or something like that? (@paked? )